SwiftIOSExperiments/Modules/Profile/View/ProfileViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ProfileViewController.swift
+//  SwiftIOSExperiments
+//
+//  Created by James Santos on 14/2/17.
+//  Copyright © 2017 James Santos. All rights reserved.
+//
+
+import UIKit
+import Auth0
+
+class ProfileViewController: UIViewController {
+	var presenter: ProfilePresentation!
+
+	@IBOutlet weak var name: UILabel!
+	@IBOutlet weak var email: UILabel!
+	@IBOutlet weak var nickName: UILabel!
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		setupView()
+		presenter.viewDidLoad()
+	}
+
+	fileprivate func setupView() {
+		name.text = ""
+		email.text = ""
+		nickName.text = ""
+
+		self.navigationItem.title = "Profile"
+		self.navigationItem.leftBarButtonItem =
+			UIBarButtonItem(title: "Logout", style: .plain, target: self, action: #selector(logout))
+	}
+
+	@objc func logout() {
+		presenter.logout()
```

Comment:
It warns me when using `logout` on this line
https://github.com/sydcanem/SwiftIOSExperiments/pull/7/files#diff-814e1e3e385aaa09eed658e9d7ca1f2cR32